pkg/minishift/clusterup/clusterup.go

```diff
@@ -29,6 +29,7 @@ import (
 	"strings"
 	"time"
 
+	startFlags "github.com/minishift/minishift/cmd/minishift/cmd/config"
```

Comment:
@budhrg suggestion: do not try to abbreviate or shorten words when not necessary.